tests/js/testUtils.js

```diff
@@ -56,4 +56,12 @@ https://github.com/gpii/universal/LICENSE.txt
         });
     };
 
+    gpii.tests.utils.simulateKeydown = function (onElement, keyCode) {
+        var eventObj = document.createEvent("Events");
+        eventObj.initEvent("keydown", true, true);
```

Comment:
Would it be possible to make this into a generic key event simulator by taking in the event name (e.g. keydown) as an argument? If so, are there other existing tests that could be refactored to use this?